roboteam_ai/src/skills/gotopos/GoAroundPos.cpp

```diff
@@ -41,104 +41,111 @@ void GoAroundPos::onInitialize() {
-    if (ballIsTarget){
-        distanceFromPoint=BALL_DIST;
+    if (ballIsTarget) {
+        distanceFromPoint = BALL_DIST;
     }
-    else{
-        if (properties->hasDouble("RotatingDistance")){
-            distanceFromPoint=properties->getDouble("RotatingDistance");
+    else {
+        if (properties->hasDouble("RotatingDistance")) {
+            distanceFromPoint = properties->getDouble("RotatingDistance");
         }
-        else{
+        else {
             ROS_ERROR_STREAM("No rotating distance set! Defaulting to ball distance");
-            distanceFromPoint=BALL_DIST;
+            distanceFromPoint = BALL_DIST;
         }
     }
     currentTick = 0;
     angleDif = Control::angleDifference(startAngle, endAngle);
     maxTick = floor(angleDif/SPEED*Constants::TICK_RATE());
-    currentProgress=ROTATING;
+    currentProgress = ROTATING;
 }
 
-
-GoAroundPos::Status GoAroundPos::onUpdate() {
+GoAroundPos::Status GoAroundPos::gtpUpdate() {
     if (! robot) {
         ROS_ERROR("Robot not found ree:  %s", std::to_string(robot->id).c_str());
         return Status::Failure;
     }
-    if (ballIsTarget && ! ball) {ROS_ERROR_STREAM("GoAroundPos update -> No ball found!");return Status::Failure; }
+    if (ballIsTarget && ! ball) {
+        ROS_ERROR_STREAM("GoAroundPos update -> No ball found!");
+        return Status::Failure;
+    }
     if (ballIsTarget) {
         targetPos = ball->pos;
     }
     if (currentTick <= maxTick) {
         commandPos =
-                targetPos + Vector2(distanceFromPoint, 0).rotate(startAngle + rotateDir*currentTick/maxTick*angleDif+M_PI);
+                targetPos + Vector2(distanceFromPoint, 0).rotate(
+                        startAngle + rotateDir*currentTick/maxTick*angleDif + M_PI);
     }
     else {
-        commandPos=targetPos+Vector2(distanceFromPoint,0).rotate(endAngle+M_PI);
+        commandPos = targetPos + Vector2(distanceFromPoint, 0).rotate(endAngle + M_PI);
     }
     deltaPos = targetPos - robot->pos;
-    currentProgress=checkProgression();
-    currentTick++;
+    currentProgress = checkProgression();
+    currentTick ++;
     // Visualization
     std::vector<std::pair<rtt::Vector2, QColor>> displayColorData;
-    displayColorData.emplace_back(std::make_pair(commandPos,Qt::red));
-    displayColorData.emplace_back(std::make_pair(targetPos + Vector2(distanceFromPoint, 0).rotate(endAngle+M_PI),Qt::red));
+    displayColorData.emplace_back(std::make_pair(commandPos, Qt::red));
+    displayColorData.emplace_back(
+            std::make_pair(targetPos + Vector2(distanceFromPoint, 0).rotate(endAngle + M_PI), Qt::red));
     interface::Drawer::setNumTreePoints(robot->id, displayColorData);
 
-    switch(currentProgress) {
-    case ROTATING: sendRotateCommand(); return Status::Running;
-    case STOPPING: sendRotateCommand(); return Status::Running;
+    switch (currentProgress) {
+    case ROTATING: sendRotateCommand();
```

Comment:
Either you fix all of them or not, don't just mindlessly control alt l because this is worse than everything being on one line.